src/huggingface_hub/commands/scan_cache.py

```diff
@@ -80,60 +80,136 @@ def run(self):
-        if self.verbosity == 0:
-            print(
-                tabulate(
-                    rows=[
-                        [
-                            repo.repo_id,
-                            repo.repo_type,
-                            "{:>12}".format(repo.size_on_disk_str),
-                            repo.nb_files,
-                            repo.last_accessed_str,
-                            repo.last_modified_str,
-                            ", ".join(sorted(repo.refs)),
-                            str(repo.repo_path),
-                        ]
-                        for repo in sorted(hf_cache_info.repos, key=lambda repo: repo.repo_path)
-                    ],
-                    headers=[
-                        "REPO ID",
-                        "REPO TYPE",
-                        "SIZE ON DISK",
-                        "NB FILES",
-                        "LAST_ACCESSED",
-                        "LAST_MODIFIED",
-                        "REFS",
-                        "LOCAL PATH",
-                    ],
-                )
-            )
-        else:
-            print(
-                tabulate(
-                    rows=[
-                        [
-                            repo.repo_id,
-                            repo.repo_type,
-                            revision.commit_hash,
-                            "{:>12}".format(revision.size_on_disk_str),
-                            revision.nb_files,
-                            revision.last_modified_str,
-                            ", ".join(sorted(revision.refs)),
-                            str(revision.snapshot_path),
-                        ]
-                        for repo in sorted(hf_cache_info.repos, key=lambda repo: repo.repo_path)
-                        for revision in sorted(repo.revisions, key=lambda revision: revision.commit_hash)
-                    ],
-                    headers=[
-                        "REPO ID",
-                        "REPO TYPE",
-                        "REVISION",
-                        "SIZE ON DISK",
-                        "NB FILES",
-                        "LAST_MODIFIED",
-                        "REFS",
-                        "LOCAL PATH",
-                    ],
-                )
-            )
+        print(get_table(hf_cache_info, verbosity=self.verbosity))
+
+
+def get_table(hf_cache_info: HFCacheInfo, *, verbosity: int = 0) -> str:
+    """Generate a table from the HFCacheInfo object.
+
+    Use `get_table(hf_cache_info, verbosity=0)` to get a table with a single row per repo, with columns
+    "repo_id", "repo_type", "size_on_disk", "nb_files", "last_accessed", "last_modified", "refs", "local_path".
+
+    Use `get_table(hf_cache_info, verbosity=1)` to get a table with a row per repo and revision (thus multiple rows can appear for a single repo), with columns
+    "repo_id", "repo_type", "revision", "size_on_disk", "nb_files", "last_modified", "refs", "local_path".
+
```

Comment:
```suggestion

    Example:
```

Adding `example` section title will be recognized as such by the doc builder